pkgs/misc/uboot/default.nix

```
@@ -108,7 +108,12 @@ let
       maintainers = with maintainers; [ dezgeg samueldr lopsided98 ];
     } // extraMeta;
   } // removeAttrs args [ "extraMeta" ]);
-
+  pi4ExtraPatches = [
+    (fetchpatch {
+      url = "https://patchwork.ozlabs.org/series/259129/mbox/";
```

Getting from the `mbox` url to the discussion page is not trivial (e.g. can't just remove `/mbox/` from the URL).

```suggestion
    # https://patchwork.ozlabs.org/project/uboot/patch/20210822143656.289891-1-sjoerd@collabora.com/
    (fetchpatch {
      url = "https://patchwork.ozlabs.org/series/259129/mbox/";
```

---

Updated with a slightly longer comment